jupyter_scheduler/pydantic_v1/__init__.py

```diff
@@ -0,0 +1,25 @@
+from importlib import metadata
+
+## Create namespaces for pydantic v1 and v2.
+# Adapted from LangChain's langchain.pydantic_v1 module.
+#
+# This code must stay at the top of the file before other modules may
+# attempt to import pydantic since it adds pydantic_v1 and pydantic_v2 to sys.modules.
+#
+# This hack is done for the following reasons:
+# * Code will attempt to remain compatible with both pydantic v1 and v2 since
+#   both dependencies and dependents may be stuck on either version of v1 or v2.
+# * Creating namespaces for pydantic v1 and v2 should allow us to write code that
+#   unambiguously uses either v1 or v2 API.
+# * This change is easier to roll out and roll back.
+
+try:
+    from pydantic.v1 import *
+except ImportError:
+    from pydantic import *
+
+
+try:
+    _PYDANTIC_MAJOR_VERSION: int = int(metadata.version("pydantic").split(".")[0])
+except metadata.PackageNotFoundError:
+    _PYDANTIC_MAJOR_VERSION = 0
```

Comment:
This can be deleted, as setting this `_PYDANTIC_MAJOR_VERSION` does nothing except expose this information to LangChain's test suite: https://github.com/search?q=repo%3Alangchain-ai/langchain%20_PYDANTIC_MAJOR_VERSION&type=code